### PR TITLE
feat(route): add 科技玩家精选

### PR DESCRIPTION
**Pull request by gkeyes**
## 该 PR 相关 Issue / Involved Issue

Close #12441, Close #12435, Close #12433, Close #12432, Close #12430, Close #12429, Close #12428, Close #12427, Close #12426

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/kejiwanjia/jingxuanpost
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note
添加科技玩家精选

### CODE REVIEW

**Comment by gkeyes** [comment]
@TonyRL Thank you! ☺️  My first PR.

File: lib/v2/kejiwanjia/radar.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'kejiwanjia.com': {
+        _name: '科技玩家精选',
+        www: [
+            {
+                title: '科技玩家精选',
```
Comment (TonyRL) [inline lib/v2/kejiwanjia/radar.js:6]:
```suggestion
        _name: '科技玩家',
        www: [
            {
                title: '精选',
```

File: lib/v2/kejiwanjia/subscribe.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                return item;
+            })
+        )
+    );
+
+    ctx.state.data = {
+        // 源标题
+        title: `科技玩家精选`,
+        // 源链接
+        link: `https://www.kejiwanjia.com/jingxuanpost`,
```
Comment (TonyRL) [inline lib/v2/kejiwanjia/subscribe.js:42]:
```suggestion
        title: '科技玩家精选',
        // 源链接
        link: 'https://www.kejiwanjia.com/jingxuanpost',
```

File: lib/v2/kejiwanjia/subscribe.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const { data: response } = await got(item.link);
+                const $ = cheerio.load(response);
+                item.description = $('.entry-content').first().html();
+                return item;
+            })
+        )
+    );
```
[the remark this answers is not in the document]
Comment (gkeyes) [reply lib/v2/kejiwanjia/subscribe.js:36]: 😂I don't really know how to do it now. I will learn it when I have time.

**Review by TonyRL — COMMENTED** [review]
**Do not** modify RSSHub core files.

**Pull request closed without merging**